spec/rails_helper.rb

```diff
@@ -45,8 +45,6 @@
   # This is also best set to 'false' when using another tool to manage the test
   # database, such as database_cleaner
 
-  # TODO: Explicitly describe the DatabaseCleaner strategies used and the reasons
```

Comment:
I made a separate issue for this [documentation request](https://github.com/EBWiki/EBWiki/issues/4027)

Reply:
That's fine.  We're going to upgrade soon (1 gem away!) and that will include the changes to system specs that allow us to ditch `database_cleaner`, so I thought we might be fine ignoring that for right now anyways.